treenode.go

```diff
@@ -126,9 +128,9 @@ func (n *TreeNodeInstance) SendTo(to *TreeNode, msg interface{}) error {
 	var c *GenericConfig
 	// only sends the config once
 	n.configMut.Lock()
-	if n.config != nil {
+	if !n.sentTo[to.ID] {
 		c = n.config
-		n.config = nil
+		n.sentTo[to.ID] = true
 	}
```

Comment:
Because the grand children are already "alive",i.e. created. THe configuration can only be passed during the *creation* of a `ProtocolInstance` in a `Service`, not during its lifetime.